app/views/users/show.html.erb

```diff
@@ -1,10 +1,21 @@
 <p style="color: green"><%= notice %></p>
 
-<%= render @user %>
-
+<div class="specific_user">
+  <%= render 'shared/user', user: @user%>
+  <div class="bio_section">
+    <h2>Bio</h2>
+    <%=@user.bio%>
+  </div>
+  <div class="post_section">
+    <h2>User Posts</h2>
+    <% @user.posts.each do |post| %>
+      <%= render 'shared/post', post: post%>
+    <% end %>
```

Comment:
- [ ] Please use the method `most_recent_post` available in the `user` model instead of iterating through all the `posts` as on the wireframe to display only 3 recent posts on the user show page.